Remotion/Web/Development.WebTesting/HostingStrategies/AspNetCoreHostingStrategy.cs

```diff
@@ -28,13 +27,18 @@ public class AspNetCoreHostingStrategy : IHostingStrategy
   /// </summary>
   /// <param name="testSiteLayoutConfiguration">The configuration of the layout of the used test site.</param>
   /// <param name="port">Port to be used.</param>
-  public AspNetCoreHostingStrategy (ITestSiteLayoutConfiguration testSiteLayoutConfiguration, int port)
+  /// <param name="hostname">Hostname to be used..</param>
+  /// <param name="useHttps">Determines whether the site is hosted using https.</param>
+  public AspNetCoreHostingStrategy (ITestSiteLayoutConfiguration testSiteLayoutConfiguration, int port, string hostname, bool useHttps)
```

Comment:
Do we have unit test for this type?

Reply:
We dont have unit tests for any hosting strategies.